crate/operator/create.py

```diff
@@ -618,7 +622,9 @@ def get_statefulset_init_containers(crate_image: str) -> List[V1Container]:
             image="busybox:1.35.0",
             image_pull_policy="IfNotPresent",
             name="init-sysctl",
-            security_context=V1SecurityContext(privileged=True),
+            security_context=V1SecurityContext(
+                capabilities=V1Capabilities(add=["SYS_ADMIN"])
```

Comment:
I had double checked on `SYS_ADMIN`, seems like it does not cut it here, looks like we need to specify:
`security_context=V1SecurityContext(privileged=True),`